### PR TITLE
Issue#95

### PR DESCRIPTION
**Pull request by NejcZupec**
Login and register links moved to the top-right .

### CODE REVIEW

File: piplmesh/frontend/static/piplmesh/css/style.css [hunk 1]
```diff
@@ -27,34 +32,65 @@ ul.errorlist {
...
+#header .user ul li.last {
+    border: none;
+}
+
+#header .user img {
+    margin-top: 5px;
+    float: right;
 }
 
+
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:82]: Why empty line?

File: piplmesh/frontend/templates/header.html [hunk 2]
```diff
@@ -2,10 +2,34 @@
...
+                <ul>
+                    <li>{% trans "Username:" %} <strong>{{ user.username }}</strong> </li>   
+                    <li class="last"> 
+                        <form method="post" action="{% url logout %}" id="logout_form">
+                            {% csrf_token %}
+                            <p><button type="submit" class="auth_button" value="value">{% trans "Log out" %}</button></p>
+                        </form>
+                    </li>
+                </ul>
+                <img src="{% facebook_picture user.username "square" %}" />
```
Comment (mitar) [inline piplmesh/frontend/templates/header.html:21]: Why image is not in `li`?

File: piplmesh/frontend/templates/header.html [hunk 3]
```diff
@@ -2,10 +2,34 @@
...
+            {% else %}
+                <ul>
+                    <li><a href="{% url login %}">{% trans "Log in" %}</a></li>
+                    <li class="last"><a href="{% url registration %}">{% trans "Register" %}</a></li>
+                </ul>
+            {% endif %}
+        </div>
+        {% comment %}
```
Comment (mitar) [inline piplmesh/frontend/templates/header.html:29]: I think this comment is unnecessary. This is evident from the template source.

**Comment by mitar** [comment]
This cannot be pulled in. Please update to the current main repository.

**Comment by NejcZupec** [comment]
Is this still relevant? I think you have already pulled in this (similar) code from another team. But If you want I can repair this pull request :)

**Comment by mitar** [comment]
You decide. :-) So if you think you can improve on existing code. Or close this pull request.

**Comment by NejcZupec** [comment]
I will try to improve it :)

**Pull request closed without merging**